### PR TITLE
Add: istag plugin 

### PR DESCRIPTION
**Pull request by devarshshah15**

### CODE REVIEW

File: velero-plugins/common/types.go [hunk 1]
```diff
@@ -1,37 +1,31 @@
 package common
-
 type routingConfig struct {
-	Subdomain string `json:"subdomain"`
```
Comment (dymurray) [inline velero-plugins/common/types.go:4]: Think we need to update this file with tabs

File: velero-plugins/common/types.go [hunk 2]
```diff
@@ -1,37 +1,31 @@
...
 type APIServerConfig struct {
-	ImagePolicyConfig imagePolicyConfig `json:"imagePolicyConfig"`
-	RoutingConfig     routingConfig     `json:"routingConfig"`
+      ImagePolicyConfig imagePolicyConfig `json:"imagePolicyConfig"`
+      RoutingConfig     routingConfig     `json:"routingConfig"`
 }
-
 const BackupServerVersion string = "openshift.io/backup-server-version"
 const RestoreServerVersion string = "openshift.io/restore-server-version"
-
```
Comment (dymurray) [inline velero-plugins/common/types.go]: I'm fine with removing some line breaks but would prefer if we were grouping the constants and breaking by lines.

File: velero-plugins/main.go [hunk 3]
```diff
@@ -53,74 +55,56 @@ func main() {
 		RegisterRestoreItemAction("openshift.io/20-SCC-restore-plugin", newSCCRestorePlugin).
 		RegisterRestoreItemAction("openshift.io/21-role-bindings-restore-plugin", newRoleBindingRestorePlugin).
 		RegisterRestoreItemAction("openshift.io/22-cluster-role-bindings-restore-plugin", newClusterRoleBindingRestorePlugin).
-
 		Serve()
 }
-
```
Comment (dymurray) [inline velero-plugins/main.go:59]: Let's leave these line breaks. Looks better in my opinion.

File: velero-plugins/common/types.go (hunk 1 above)
Comment (devarshshah15) [reply velero-plugins/common/types.go:4]: Ok, will try to update it once again. It already is with tabs based on the setting but will commit it again.

File: velero-plugins/main.go (hunk 3 above)
Comment (devarshshah15) [reply velero-plugins/main.go:59]: So, need to remove line break, right?

Comment (dymurray) [reply velero-plugins/main.go:59]: In this file let's add the line breaks back in. For some reason they were removed here

Comment (devarshshah15) [reply velero-plugins/main.go:59]: Added

**Review by dymurray — APPROVED** [review]

**Pull request merged**